hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/StreamSync.java

```diff
@@ -523,9 +526,15 @@ private Pair<SchemaProvider, Pair<String, JavaRDD<HoodieRecord>>> fetchFromSourc
       throw new UnsupportedOperationException("Spark record only support parquet log.");
     }
 
+    HoodieTableMetaClient metaClient = HoodieTableMetaClient.builder().setConf(new Configuration(fs.getConf()))
```

Comment:
syncOnce() impl already has an instance of HoodieTableMetaClient just after refreshing the timeline. we should try to re-use that instead of creating new ones.